packages/dd-trace/src/appsec/remote_config/manager.js

```diff
@@ -256,16 +250,42 @@ class RemoteConfigManager extends EventEmitter {
 
       // in case the item was already handled by kPreUpdate
       if (item.apply_state === UNACKNOWLEDGED || action === 'unapply') {
-        try {
-          // TODO: do we want to pass old and new config ?
-          const hadListeners = this.emit(item.product, action, item.file, item.id)
-
-          if (hadListeners) {
-            item.apply_state = ACKNOWLEDGED
+        const handler = this._handlers.get(item.product)
+
+        if (handler) {
```

Comment:
wdyt about a continue to avoid an indentation level ?
```suggestion
        if (!handler) continue
```
or you could also have a `_handlers.has(item.product)` directly in the if case above

Reply:
> wdyt about a continue to avoid an indentation level ?

I can't use `continue` as that would skip the last part of the the for loop, not only the nested if-statement.

> or you could also have a _handlers.has(item.product) directly in the if case above

I need the actual `handler` function so that I can call it inside the if-statement, so using `has()` would be an unnecessary function-call as I already have retrieved the `handler` a few lines above.

Reply:
Irrelevant as the new code doesn't have the problem anymore